src/assessment.js

```diff
@@ -19,7 +19,7 @@
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
+var q1 = /* your answer here */;false
```

Comment:
Sorry, the answer here is `true`. A line following `var fire = function fire(){};` will not be inside the `fire` function; as such it will remain in the `deathStar` scope.